src/http.rs

```diff
@@ -401,6 +401,10 @@ pub struct KeepaliveConfig {
     /// The maximum amount of time a connection may exist before it is closed
     /// by sending a `Connection: close` header on the HTTP response.
     ///
+    ///
+    /// Set this to a large value like `100000000` to disable this feature (in the future setting
```

Comment:
```suggestion
    /// Set this to a large value like `100000000` to disable this feature (in the future, setting
```
We generally avoid these kinds of "future" statements in the docs.

Reply:
Thanks, agreed it does seem prudent to avoid these sorts of future statements since they may never come to be.